tests/cargo-upgrade/main.rs

```diff
@@ -146,6 +146,14 @@ fn add_everything_registry_packages(alt: bool) {
     }
 }
 
+fn add_git_registry_packages() {
+    cargo_test_support::git::new("serde", |project| {
+        project
+            .file("Cargo.toml", &cargo_test_support::basic_manifest("serde", "1.0.99999"))
```

Comment:
It might be more in line with the other tests to have the patch number be lower (i.e. not 99999). Not sure what the convention was here.